examples/android/src/main/AndroidManifest.xml

```diff
@@ -2,10 +2,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.alohalytics.demoapp" >
 
-    <!-- This permission is needed if you want to access IMEI -->
-    <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
-    <!-- This one is needed to upload statistics to the server -->
+    <!-- REQUIRED to upload statistics to the server -->
```

Comment:
Here they are parsed by humans only who will study an example. They will be mentioned in final documentation too. In real life they are parsed by eyes of each user who wants to install the app on the market. And after that they are parsed by the runtime to allow or disallow access to some system APIs.